portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+    </nav>
+    <div class="heading">
+        <h1>My Projects</h1>
+    </div>
+   <div id="projects-content">
+    <div id="content">
```

Comment:
I'm also seeing duplicate IDs in this file and in gallery.html. Would you mind updating them as well?

Reply:
@pruess-google Fixed those too!